modules/edit.js

```diff
@@ -0,0 +1,25 @@
+import fetchLocal from './fetchLocal.js';
+
+export default () => {
+  const listul = document.querySelector('.list-ul');
+  let tasklist;
+  listul.addEventListener('keydown', (event) => {
+    tasklist = fetchLocal();
+
+    if (event.target.classList.contains('description')) {
+      const listid = event.target.id;
+      const edittask = document.getElementById(listid);
+      // edittask.setAttribute("contenteditable", "true");
```

Comment:
- [ ] Please do not commit unused code to your repository because it makes your code look messy so please kindly remove all the commented code in your work

Reply:
Thanks for reviewing, I almost forget to remove 👍